index.html

```diff
@@ -90,8 +99,21 @@ <h2>Web design is my passion </h2>
         </div>
         Hi
         <br>
-        <a class="btn btn-primary" onclick="Fortune()">Lucky luck luck</a>
-        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-9-circle" viewBox="0 0 16 16">
+        <a class="btn btn-primary" onclick="Fortune()">Lucky luck luck</a> 
+        <br>
+
+        <h1>Tarot Lesung</h1>
+        <div id="card1" class="card"><img src="" alt="Karte 1"></div>
+        <div id="card2" class="card"><img src="" alt="Karte 2"></div>
+        <div id="card3" class="card"><img src="" alt="Karte 3"></div>
+        <br><br><br>
+        <button onclick="drawCards()">Ziehe Karten</button>
```

Comment:
no bootstrap? (buttons without styles look pretty bad on any browser other than firefox and safari)